### ISSUE TITLE
Did you train on the training set and validation set both and then test on the test set?

### ISSUE THREAD

**Issue by zhaobinNF** [sel118/LaneAF#33]
Your paper is very wonderful. But I still have some confusions. Did you train on the training set and validation set both and then test on the test set? Or just trained on the training set then test on the testing set?

**Comment by arangesh** [comment]
All three datasets we train/evaluate on provides a separate train, val and test set. We use only the training set for training and monitor the metrics on the validation set. The results are then reported on the test set.
